### PR TITLE
Add: Toast Component

### PR DESCRIPTION
**Pull request by devsurajrai**
Added the toast component to my component library. Please Review.

toast

### CODE REVIEW

File: components/toasts/toasts.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* reset  */
+.toast-text{
+    padding:unset;
+    margin:unset;
```
Comment (Dev-AkshatGupta) [inline components/toasts/toasts.css:4]: Nice use to mute the pre-defined styles

**Review by Dev-AkshatGupta — APPROVED** [review]
Seems good to me changes have nothing to suggest

File: components/toasts/toasts.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <iframe class="iframe-codes cdn-link"
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%252F%252Fadd%2520this%2520link%2520in%2520the%2520hader%2520section%2520of%2520the%2520HTML%2520file.%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fui-phoenix.netlify.app%252Fcomponents%252Ftoasts%252Ftoasts.css%2522%253E%250A%2520%2520%252F%252F%2520add%2520this%2520script%2520file%2520at%2520the%2520of%2520the%2520body%2520%28after%2520footer%29.%2520just%2520before%2520the%2520body%2520closing%2520tag.%250A%253Cscript%2520src%253D%2522https%253A%252F%252Fui-phoenix.netlify.app%252Fcomponents%252Ftoasts%252Ftoasts.js%2522%253E%253C%252Fscript%253E"
+                style="width: 942px; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+
+            </iframe>
+
+        </div>
+        <div class="modal-parent-container">
+            <h2></h2>
```
Comment (SJTGSHIVAM) [inline components/toasts/toasts.html:32]: Just asking is it a filler? I guess yes.

File: shared/style-guide.css [hunk 3]
```diff
@@ -24,6 +24,7 @@
   --online: green;
```
Comment (SJTGSHIVAM) [inline shared/style-guide.css:24]: clean way to handle things I must say.

**Review by SJTGSHIVAM — APPROVED** [review]
Small PRs have this benefit that there is not much to discuss. I would say we are good to go.

**Review by saifudinbadani — APPROVED** [review]
The code was easy to go through , no issue from my side.

**Pull request merged**